slipp/src/main/java/slipp/controller/ApiUserController.java

```diff
@@ -58,8 +61,10 @@ public ModelAndView update(HttpServletRequest request, HttpServletResponse respo
         UserUpdatedDto updateDto = objectMapper.readValue(request.getInputStream(), UserUpdatedDto.class);
         logger.debug("Updated User : {}", updateDto);
 
-        User user = DataBase.findUserById(userId);
+        User user = userDao.findByUserId(userId)
+                .orElseThrow(() -> new IllegalArgumentException("유저가 없어요."));
```

Comment:
GET API와 동일하게 UserNotFoundException 사용해도 될 것 같아요.
```suggestion
                .orElseThrow(UserNotFoundException::new);
```

Reply:
앗, 처음에 ``IllegalArgumentException`` 으로 해두고 바꾼다는 게, 바꾸다가 빼먹은 것 같아요 ! 감사합니다.